locales/en/breach-checklist.ftl

```diff
@@ -0,0 +1,67 @@
+
+## Prompts the user for changes when there is a breach detected of social security number
+
+breach-checklist-ssn-header = Monitor your credit report for accounts, loans, or credit cards you don’t recognize.
+breach-checklist-ssn-body = You can also consider freezing your credit on <a>Equifax</a>, <a>Experian</a> and <a>TransUnion</a> to stop scammers from opening new accounts in your name. It’s free and won’t affect your credit score.
+
+## Prompts the user for changes when there is a breach detected of credit card
+
+breach-checklist-cc-header = Report this breach to your credit card issuer and request a new card with a new number.
+breach-checklist-cc-body = You should also review your credit card statements for unrecognized charges.
+
+## Prompts the user for changes when there is a breach detected of bank account
+
+breach-checklist-bank-header = Notify your bank immediately that your account number has been compromised.
+breach-checklist-bank-body = Doing so faster could give you more legal protections to help you recover any losses. You’ll also want to check your accounts for any unrecognized charges.
+
+## Prompts the user for changes when there is a breach detected of pin
+
+breach-checklist-pin-header = Notify your card issuer and change your PIN immediately.
+breach-checklist-pin-body = Make sure your new PIN, or any other PIN, doesn’t include easily guessed numbers such as your birth date or address.
+
+## Prompts the user for changes when there is a breach detected of IP address
+
+breach-checklist-ip-header = Use the internet privately with a VPN, such as <a>{-brand-mozilla-vpn}</a>.
+breach-checklist-ip-body = Your IP address (Internet Protocol address) pinpoints your location and internet service provider. A VPN can hide your real IP address so you can use the internet privately.   
+
+## Prompts the user for changes when there is a breach detected of physical address
+
+breach-checklist-address-header = Change any passwords or PINs that include any part of your address.   
+breach-checklist-address-body = Addresses are easy to find in public records and can make those passwords and PINs easy to guess.   
+
+## Prompts the user for changes when there is a breach detected of date of birth
+
+breach-checklist-dob-header = Change any passwords or PINs that include your date of birth.  
+breach-checklist-dob-body = Birth dates are easy to find in public records, and people who find it could easily guess your PIN.
+
+## Prompts the user for changes when there is a breach detected of phone number
+
+breach-checklist-phone-header = Protect your phone number with a masking service like <a>{-brand-relay}</a>, which hides your true phone number. 
+
+## Prompts the user for changes when there is a breach detected of security questions
+
+# NOTE: { $companyLink } is a placeholder for the URL to the password change page of the company where the breach occurred 
```

Comment:
It's not the link to the password page though? Should this be `$companyName` instead?

Reply:
According to copy, this should be the website. Updated the var name and comment to better reflect intent